### PR TITLE
Rebranding of plugin

### PR DESCRIPTION
**Pull request by nastra**
This updates the plugin name and the internal packaging name and prepares the plugin for a 1.8.0 release

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 # gradle-revapi
 
+This is a fork of the well-known `gradle-revapi` plugin from https://github.com/palantir/gradle-revapi and has been moved under the `revapi` umbrella.
```
Comment (ajantha-bhat) [inline README.md:7]: Can you please add "For more details refer: https://github.com/revapi/revapi/issues/296"?

**Review by ajantha-bhat — APPROVED** [review]
LGTM

File: build.gradle [hunk 2]
```diff
@@ -25,8 +25,10 @@ repositories {
     mavenCentral() { metadataSources { mavenPom(); ignoreGradleMetadataRedirection() } }
 }
 
-group 'com.palantir.gradle.revapi'
-version System.env.CIRCLE_TAG ?: gitVersion()
+group 'org.revapi.gradle.revapi'
```
Comment (metlos) [inline build.gradle:28]: what about group being just `org.revapi` like the maven artifacts? or if that's not a common convention, `org.revapi.gradle`? I find the two `revapi`s just in the group id kinda weird.

Comment (nastra) [reply build.gradle]: makes sense, I've udpated this to `org.revapi`

**Review by metlos — APPROVED** [review]

**Pull request merged**